verilog/formatting/tree_unwrapper.cc

```diff
@@ -1454,13 +1575,26 @@ void TreeUnwrapper::ReshapeTokenPartitions(
     case NodeEnum::kContinuousAssignmentStatement: {  // e.g. assign a=0, b=2;
       // TODO(fangism): group into above similar assignment statement cases?
       //   Cannot easily move now due to multiple-assignments.
+      const bool anyAppendFittingSubpartitions = std::find_if(
```

Comment:
Just realized there's `std::any_of` for this, instead of `find_if`.
https://en.cppreference.com/w/cpp/algorithm/all_any_none_of

As for the policy itself `kAppendFittingSubpartitions` might be ok, but the bigger question is: does it need to be handled in this PR, which is focused on method and function calls?